api/collection_submissions_actions/serializers.py

```diff
@@ -0,0 +1,30 @@
+from api.actions.serializers import BaseActionSerializer
+from osf.utils import permissions
+from osf.utils.workflows import CollectionSubmissionsTriggers
+from rest_framework import serializers as ser
+from api.actions.serializers import TargetRelationshipField
+from osf.models import CollectionSubmission
+from osf.utils.workflows import ApprovalStates
+
+
+class CollectionSubmissionActionSerializer(BaseActionSerializer):
```

Comment:
Please add a note describing inherited fields for easy reference

Reply:
Since there were some more inheirtance problems I just decoupled this from `BaseActionSerializer`. so the fields are more explicit.